### ISSUE TITLE
FreeType on Windows VS 2022: ModuleNotFoundError: No module named 'mesonbuild.dependencies.dev'

### ISSUE THREAD

**Issue by Demonese** [mesonbuild/meson#12068]
**Describe the bug**

```
PS C:\Users\Admin\Desktop\freetype-2.13.1> meson setup build
The Meson build system
Version: 1.2.0
Source dir: C:\Users\Admin\Desktop\freetype-2.13.1
Build dir: C:\Users\Admin\Desktop\freetype-2.13.1\build
Build type: native build
Project name: freetype2
Project version: 2.13.1
Activating VS 17.6.5
C compiler for the host machine: cl (msvc 19.36.32537 "Microsoft (R) C/C++ Optimizing Compiler Version 19.36.32537 for x64")
C linker for the host machine: link link 14.36.32537.0
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program python3 found: YES
Has header "unistd.h" : NO 
Has header "fcntl.h" : YES 
Has header "sys/mman.h" : NO 
Windows resource compiler: Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384
Traceback (most recent call last):
  File "mesonbuild\mesonmain.py", line 194, in run
  File "mesonbuild\msetup.py", line 358, in run
  File "mesonbuild\msetup.py", line 183, in generate
  File "mesonbuild\msetup.py", line 228, in _generate
  File "mesonbuild\interpreter\interpreter.py", line 2997, in run
  File "mesonbuild\interpreterbase\interpreterbase.py", line 164, in run
  File "mesonbuild\interpreterbase\interpreterbase.py", line 190, in evaluate_codeblock
  File "mesonbuild\interpreterbase\interpreterbase.py", line 182, in evaluate_codeblock
  File "mesonbuild\interpreterbase\interpreterbase.py", line 206, in evaluate_statement
  File "mesonbuild\interpreterbase\interpreterbase.py", line 301, in evaluate_if
  File "mesonbuild\interpreterbase\interpreterbase.py", line 190, in evaluate_codeblock
  File "mesonbuild\interpreterbase\interpreterbase.py", line 182, in evaluate_codeblock
  File "mesonbuild\interpreterbase\interpreterbase.py", line 198, in evaluate_statement
  File "mesonbuild\interpreterbase\interpreterbase.py", line 635, in assignment
  File "mesonbuild\interpreterbase\interpreterbase.py", line 196, in evaluate_statement
  File "mesonbuild\interpreterbase\interpreterbase.py", line 521, in function_call
  File "mesonbuild\interpreterbase\decorators.py", line 826, in wrapped
  File "mesonbuild\interpreterbase\decorators.py", line 826, in wrapped
  File "mesonbuild\interpreterbase\decorators.py", line 826, in wrapped
  [Previous line repeated 4 more times]
  File "mesonbuild\interpreterbase\decorators.py", line 109, in wrapped
  File "mesonbuild\interpreterbase\decorators.py", line 127, in wrapped
  File "mesonbuild\interpreterbase\decorators.py", line 277, in wrapper
  File "mesonbuild\interpreterbase\decorators.py", line 596, in wrapper
  File "mesonbuild\interpreter\interpreter.py", line 1774, in func_dependency
  File "mesonbuild\interpreter\dependencyfallbacks.py", line 355, in lookup
  File "mesonbuild\interpreter\dependencyfallbacks.py", line 87, in _do_dependency
  File "mesonbuild\dependencies\detect.py", line 115, in find_external_dependency
  File "mesonbuild\dependencies\detect.py", line 193, in _build_external_dependency_list
  File "collections\__init__.py", line 1123, in __getitem__
  File "mesonbuild\dependencies\detect.py", line 39, in __missing__
  File "importlib\__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'mesonbuild.dependencies.dev'

meson.build:285:13: ERROR: Unhandled python exception

    This is a Meson bug and should be reported!
```

**To Reproduce**

* Download Freetype 2.13.1 from https://freetype.org/download.html  
* Decompress  
* `meson setup build`

**Expected behavior**

No errors thrown by meson itself.

**system parameters**
* Not - Is this a [cross build](https://mesonbuild.com/Cross-compilation.html) or just a plain native build (for the same computer)?
* Windows 11 22H2 - what operating system (e.g. MacOS Catalina, Windows 10, CentOS 8.0, Ubuntu 18.04, etc.)
* No Python installed (msi meson installer) - what Python version are you using e.g. 3.8.0
* 1.2.0 - what `meson --version`
* 1.11.1 - what `ninja --version` if it's a Ninja build

**Comment by eli-schwartz** [comment]
Duplicate of #12036

**Comment by eli-schwartz** [comment]
> No Python installed (msi meson installer) - what Python version are you using e.g. 3.8.0

This is a known issue specifically in the MSI installer that will be fixed in the upcoming 1.2.1 bugfix release.

Until then, you can use [the 1.1 release](https://github.com/mesonbuild/meson/releases/tag/1.1.1).

(Or you can install python, and then use one of the non-MSI installation methods from https://mesonbuild.com/Getting-meson.html, if you really want.)

**Comment by eli-schwartz** [comment]
The bugfix release is out now, please give the new `meson-1.2.1-64.msi` a try.
